XpringKit.xcodeproj/project.pbxproj

```diff
@@ -1551,8 +1562,6 @@
 			isa = PBXProject;
 			attributes = {
 				LastUpgradeCheck = 1020;
-				TargetAttributes = {
-				};
```

Comment:
out of curiosity, was this just unused from a previous something?

Reply:
This file is autogenerated when you run `xcodegen`. 

It's likely that this changes based on what exact version of XCode you have installed. The file which generates this file is `project.yml` and that's the canonical definition.